lib/mix/tasks/migrate_db.ex

```diff
@@ -0,0 +1,80 @@
+defmodule Mix.Tasks.MigrateDb do
+  use Mix.Task
+
+  require Ex2ms
+
+  alias AeMdw.Db.Model
+  alias AeMdw.Db.Util
+  alias AeMdw.Log
+
+  @table Model.Migrations
+  @record_name Model.record(@table)
+  @version_len 14
+
+  @migrations_code_path "priv/migrations/*.ex"
+
+  @impl Mix.Task
+  def run(_) do
+    :net_kernel.start([:aeternity@localhost, :shortnames])
+    :ae_plugin_utils.start_aecore()
+    :lager.set_loglevel(:epoch_sync_lager_event, :lager_console_backend, :undefined, :error)
+    :lager.set_loglevel(:lager_console_backend, :error)
+    Log.info("================================================================================")
+    current_version = read_migration_version(@table)
```

Comment:
`@table` is accessible everywhere, why pass it around?

Reply:
I usually prefer making it explicit what the functions is depending on but good `read_migration_version` was enough.